VMD_scripts/blobulation.tcl

```diff
@@ -89,23 +89,25 @@ proc ::blobulator::blobulate {MolID lMin H dictInput} {
 
 		} else {
 		
-	set sequence [::blobulator::getSequence $MolID]
-	set hydroS [::blobulator::hydropathyScores $usedDictionary $sequence]
-	if {$hydroS == -1} {
-		return -1
-		}
-	set smoothHydro [::blobulator::hydropathyMean $hydroS $sequence]
-	set digitized [::blobulator::digitize $H $smoothHydro ]
-	set hblob [ ::blobulator::hBlob $digitized $lMin ]
-	set hsblob [ ::blobulator::hsBlob  $hblob $digitized $lMin ]
-	set hpsblob [ ::blobulator::hpsBlob  $hsblob $digitized ]
-	set groupedBlob [::blobulator::blobGroup $hpsblob]
-	set blobulated [::blobulator::blobAssign $hpsblob]
-	set blobIndexList [ ::blobulator::blobIndex $blobulated ]
-	if {$blobulated != -1} {
-		::blobulator::blobUserAssign $blobulated $MolID
-		::blobulator::blobUser2Assign $blobIndexList $MolID
-		blobUser3Assign $groupedBlob $MolID
+
```

Comment:
What's the purpose of these changes?

Reply:
referring to the block of changes that start on this line

Reply:
I believe no changes actually happened, I think I cut by accident and pasted directly after.

Reply:
AH it was a whitespace change - you indented.